src/components/v2/ProgressBar/AccountHealth/index.tsx

```diff
@@ -0,0 +1,132 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import Typography from '@mui/material/Typography';
+
+import PLACEHOLDER_KEY from 'constants/placeholderKey';
+import { formatCentsToReadableValue, formatToReadablePercentage } from 'utilities/common';
+import { useTranslation } from 'translation';
+import { ProgressBar } from '..';
+import { useStyles } from './styles';
+
+export interface IAccountHealthProps {
+  borrowBalanceCents: number | undefined;
+  borrowLimitCents: number | undefined;
+  safeBorrowLimitPercentage: number;
+  variant?: 'primary' | 'secondary';
+  className?: string;
+}
+
+export const AccountHealth: React.FC<IAccountHealthProps> = ({
+  className,
+  borrowBalanceCents,
+  borrowLimitCents,
+  variant = 'primary',
+  safeBorrowLimitPercentage,
+}) => {
+  const styles = useStyles();
+  const { t, Trans } = useTranslation();
+
+  let borrowLimitUsedPercentage: number | undefined;
+  if (borrowLimitCents === 0) {
+    borrowLimitUsedPercentage = 0;
+  } else if (typeof borrowBalanceCents === 'number' && typeof borrowLimitCents === 'number') {
+    borrowLimitUsedPercentage = Math.round((borrowBalanceCents * 100) / borrowLimitCents);
+  }
+
+  const readableBorrowLimitUsedPercentage = formatToReadablePercentage(borrowLimitUsedPercentage);
+
+  const safeBorrowLimitCents =
+    typeof borrowLimitCents === 'number'
+      ? Math.floor((borrowLimitCents * safeBorrowLimitPercentage) / 100)
+      : undefined;
+
+  const readableSafeBorrowLimit = formatCentsToReadableValue({
+    value: safeBorrowLimitCents,
+    removeDecimals: true,
+  });
+
+  const readableBorrowLimit = formatCentsToReadableValue({
+    value: borrowLimitCents,
+    removeDecimals: true,
+  });
+
+  const readableBorrowBalance = formatCentsToReadableValue({
+    value: borrowBalanceCents,
+  });
+
+  return (
+    <div className={className}>
+      <div css={styles.topProgressBarLegend}>
+        <div css={styles.inlineContainer}>
+          <Typography component="span" variant="small2" css={styles.inlineLabel}>
+            {variant === 'primary'
```

Comment:
I think this component would be more flexible if we dropped the variant and made these props.
If we keep the variant, I would label it more descriptive such as 'borrow' and 'supply'

Reply:
I'd rather keep it as strict as possible, since these are the only two existing variants.
"borrow" and "supply" wouldn't describe what this actually is. I tried to come up with names that would make sense, but it's actually quite tricky ahah
One variant displays the borrow limit used, the other one display the borrow balance in dollars. Maybe the variants could be named "borrowLimitUsed" and "borrowBalance"?

Reply:
"borrowLimitUsed" and "borrowBalance"  work for me